### ISSUE TITLE
Yet another occurance of tail-layering issues[GENERAL] 

### ISSUE THREAD

**Issue by BonniePandora** [VOREStation/VOREStation#13400]
#### Brief description of the bug
When viewing the character preview of the chara-setup screen, any tails the character might have aren't overlaid atop clothing, instead ending up partially hidden beneath it, most notable with the North-facing sprite.
This was found on various custom species characters of mine, using the human sprite base.

#### What you expected to happen
For the tails to display correctly, actually showing over clothing

#### What actually happened
The tails on various characters of mine were partially hidden beneath their clothing. Toggling the visibility of both job items & loadouts showed that the tail still displayed over the body, but the clothing & equipment was layered higher than the tail was.
![image](https://user-images.githubusercontent.com/24844135/181087920-b096eac7-b99f-44fb-b89e-5cdd49cd51ae.png)

#### Steps to reproduce
- Go to character setup
- Equip a character with a tail, then either assign an occupation to high or add a universally allowed uniform in their loadout
- Return to the main chara setup screen, only to see part of the chosen tail covered up by the uniform,

#### Code Revision
- Server revision: B:release D:
Commit: [9b9f3c9eeaa0beb2e7c3865620f134bde7590de0](http://127.0.0.1:64942/tmp97324/%3Ca%20href=%22https://github.com/VOREStation/VOREStation/commit/9b9f3c9eeaa0beb2e7c3865620f134bde7590de0%22%3Ehttps://github.com/VOREStation/VOREStation/commit/9b9f3c9eeaa0beb2e7c3865620f134bde7590de0%3C/a%3E)

**Comment by Heroman3003** [comment]
Yeah, the preview icons are generated differently and I tried to fix them, but ultimately couldnt really figure anything that worked consistently.
